src/components/UI/molecules/pools/bond/splitted-card.jsx

```diff
@@ -20,7 +20,7 @@ export const SplitedDetail = ({
       </h6>
       <p
         className={classNames(
-          "text-lg font-normal font-numbers whitespace-nowrap",
+          "text-lg font-normal font-numbers whitespace-normal",
```

Comment:
```suggestion
          "text-lg font-normal font-numbers",
```
Remove redundant class name

Reply:
Done.